src/test/java/com/kaltura/client/test/tests/servicesTests/productPriceTests/ListTests.java

```diff
@@ -58,24 +62,27 @@ public void ppv() {
         /*Ppv ppv = IngestPPVUtils.ingestPPV(INGEST_ACTION_INSERT, true, "My ingest PPV", getProperty(FIFTY_PERCENTS_ILS_DISCOUNT_NAME),
                 Double.valueOf(getProperty(AMOUNT_4_99_EUR)), CURRENCY_EUR, getProperty(ONE_DAY_USAGE_MODULE), false, false,
                 getProperty(DEFAULT_PRODUCT_CODE), getProperty(WEB_FILE_TYPE), getProperty(MOBILE_FILE_TYPE));*/
-        Response<ListResponse<Entitlement>> entitlementListBeforePurchase = EntitlementServiceImpl.list(sharedMasterUserKs, entitlementPpvsFilter, null);
+        client.setKs(sharedMasterUserKs);
+//        client.setCurrency("?");
```

Comment:
can be removed?

Reply:
Actually this is question for you. Do you use somewhere currency? I just changed your functionality.

Reply:
Not in that test case

Reply:
Ok so ill remove